src/main/java/com/wiley/autotest/selenium/AbstractSeleniumTest.java

```diff
@@ -129,8 +130,10 @@ public void afterMethod() {
 
     @AfterMethod(alwaysRun = true)
     public void doAfterMethods() {
-        //TODO VE: sometimes cookies may be needed so it's worth to make it optional/configurable
+        //TODO VE: sometimes local data may be needed so it's worth to make it optional/configurable
         cookiesService.deleteAllCookies();
+        ((JavascriptExecutor)getWebDriver()).executeScript("window.localStorage.clear();");
```

Comment:
@skopanirov what will happen with this code if there is no window? (i.e. it was closed). If there will be any error in this case - u need to handle it in code.

Reply:
Fixed